active/social-media.md

```diff
@@ -0,0 +1,81 @@
+# Social Media Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official social media profiles on behalf of the Django Software Foundation.
+
+The goals of the group are:
+- Promote the use of Django among the World Wide Web development community.
+- Protect the framework's long-term viability through campaigns and sponsor relations.
+- Ensure the DSF’s Code of Conduct is maintained on its media channels.
+- Assist the board in posting official news and updates from the DSF.
+
+The group opts to manage platforms that best achieve these goals.
+This may be a subset of those for which Django has accounts.
+
+Delegated responsibilities:
+- The Chair can grant access to Django's official social media profiles and each member will have access.
```

Comment:
I would recommend two changes:

- Rather than just the Chair, extend this responsibility to the Co-Chair and Board Liaison. Less bottleneck.
- Rather than mandating all members having access, leave some room to figure out who has access to what based on what’s practical. Not all platforms have the same amount of capabilities to manage shared accounts.

```suggestion
- The Chair, Co-Chair and Board Liaison can grant access to Django's official social media profiles to other group members as needed.
```